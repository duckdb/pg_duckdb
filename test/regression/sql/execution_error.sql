CREATE TABLE int_as_varchar(a varchar);
INSERT INTO int_as_varchar SELECT * from (
	VALUES
		('abc')
) t(a);

SELECT a::INTEGER FROM int_as_varchar;
DROP TABLE int_as_varchar;
