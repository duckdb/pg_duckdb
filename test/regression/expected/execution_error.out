create table int_as_varchar(a varchar);
insert into int_as_varchar SELECT * from (
	VALUES
		('abc')
) t(a);
select a::INTEGER from int_as_varchar;
ERROR:  (PGDuckDB/ExecuteQuery) Conversion Error: Could not convert string 'abc' to INT32
LINE 1: SELECT (a)::integer AS a FROM int_as_varchar
                  ^
